packages/layout/src/components/cards/common/views/remove/reason/reason.tsx

```diff
@@ -39,31 +37,42 @@ export const Reason: React.FC<ReasonProps> = ({
 				}}
 			>
 				{({ handleSubmit, submitError, form }) => {
-					const leftScheme =
-						form.getState().values.reason === 'left_the_scheme';
+					const reason = form.getState().values.reason;
+					const showError: boolean = !!submitError && !reason;
+					const leftScheme: boolean = reason === 'left_the_scheme';
 					return (
 						<form onSubmit={handleSubmit} data-testid={`remove-${type}-form`}>
-							<FFRadioButton
-								name="reason"
-								type="radio"
-								label={i18nRemoveReason.fields.leftTheScheme.label}
-								value="left_the_scheme"
-								cfg={{ my: 4 }}
-							/>
-							{leftScheme && (
-								<div className={styles.dateWrapper}>
-									{renderFields(dateField)}
-								</div>
-							)}
-							<FFRadioButton
-								name="reason"
-								type="radio"
-								label={i18nRemoveReason.fields.neverPartOfTheScheme.label}
-								value="not_part_of_scheme"
-							/>
-							{submitError && (
-								<P cfg={{ color: 'danger.2', mt: 5 }}>{submitError}</P>
-							)}
+							<div className={showError && elementStyles.labelError}>
+								<H4 fontWeight="bold" mb={0}>
```

Comment:
Is a H4 the right choice here given bug #80620?  It may be, or you can use a regular span and style it to look like a H4 if not.

Reply:
It was existing code, and probably out of scope for this ticket. That said, if it is going to be a header, it is the appropriate one. Accessibility score 100 in lighthouse.

Reply:
@allanthompson-tpr @dotcom9 When you have a heading for a group of radio buttons or checkboxes, that should be a `<fieldset>` with a `<legend>` element. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Reply:
@allanthompson-tpr @dotcom9 I've created bug AB#82897